### PR TITLE
fix: compilation_errors

### PR DESCRIPTION
**Pull request by mohitpubnub**
fix: script compilation error with Unity 2020

Fixes issue of compilation error with Unity 2020.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -48,14 +48,14 @@ jobs:
       - name: Install Unity engine
         uses: ./.github/.release/actions/actions/installers/unity
         with:
-          unity-version: "2020.3.22f1"
+          unity-version: "2022.3.11f1"
```
Comment (budgetpreneur) [inline .github/workflows/release.yml:51]: @parfeon  says on GH we have `Unity Editor version:    2020.3.22f1 (e1a7f79fd887)`, but new changes going with 11f1?

Comment (budgetpreneur) [reply .github/workflows/release.yml:51]: If we want to go with `2022.3.11f1`, then should we have a job or step in yml file to install the version we want?

Comment (parfeon) [reply .github/workflows/release.yml:51]: @mohitpubnub this won't work because hash is not in the list which I shared

Comment (parfeon) [reply .github/workflows/release.yml:51]: @mohitpubnub I'll need some time to add more versions

**Comment by mohitpubnub** [comment]
created PR https://github.com/pubnub/unity/pull/94 for fixing release

**Pull request closed without merging**